src/test/groovy/io/repaint/maven/tiles/TilesMavenLifecycleParticipantTest.groovy

```diff
@@ -7,7 +7,7 @@
  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
  * the License. You may obtain a copy of the License at
  *
- *	 http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Should be squashed into original commit.

Reply:
Pardon the stupid question, but how do I squash into the original commit? Do I squash into a new branch and submit as a new pr?

Reply:
nevermind